src/sinks/gcp/chronicle_unstructured.rs

```diff
@@ -452,6 +526,10 @@ impl ChronicleRequestBuilder {
         let encoder = ChronicleEncoder {
             customer_id: config.customer_id.clone(),
             namespace: config.namespace.clone(),
+            labels: config
+                .labels
+                .clone()
+                .map(|labels| labels.into_iter().collect_vec()),
```

Comment:
It's come to light that this need some work. We need:
```
  "labels" : [
    {
      "key" : "source",
      "value" : "vector"
    },
    {
      "key" : "key_name_two",
      "value" : "value_two"
    }
  ]
```
see https://cloud.google.com/chronicle/docs/reference/ingestion-api#request_body_2

note: This example has underscores in it?